lib/shared/snackbar.dart

```diff
@@ -28,30 +28,33 @@ void showSnackbar(
       (context, progress) {
         return SnackbarNotification(
           builder: (context) => ThunderSnackbar(
-            content: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                if (leadingIcon != null) Icon(leadingIcon, color: leadingIconColor),
-                if (leadingIcon != null) const SizedBox(width: 8.0),
-                Expanded(child: Text(text)),
-                if (trailingIcon != null)
-                  SizedBox(
-                    height: 20,
-                    child: IconButton(
-                      visualDensity: VisualDensity.compact,
-                      onPressed: trailingAction != null
-                          ? () {
-                              OverlaySupportEntry.of(context)?.dismiss();
-                              trailingAction();
-                            }
-                          : null,
-                      icon: Icon(
-                        trailingIcon,
-                        color: trailingIconColor ?? Theme.of(context).colorScheme.inversePrimary,
+            content: GestureDetector(
+              onTap: () => OverlaySupportEntry.of(context)?.dismiss(),

```

Comment:
Does this dismiss snackbars on tap? I wonder if we should have an X button for that.

<img width="1024" alt="unnamed" src="https://github.com/thunder-app/thunder/assets/7417301/088ce96e-d0d6-427d-b9c4-a53897df6cc9">

https://m3.material.io/components/snackbar/specs

Reply:
Good call!

I'll switch the approach here - the original intention was to have a way to dismiss the snackbar without swiping (while having an action present). This was because the "restored comment from draft" snackbar was annoying, and blocked the markdown toolbar entirely.

Reply:
Done in https://github.com/thunder-app/thunder/pull/1165/commits/c4fb8c4d07363d5a32b336a72481c429ab749e5e!